src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
```

Comment:
Node should be an inner class within MyLinkedList. Move the Node class inside the MyLinkedList class and make it private.